src/websocket-client.ts

```diff
@@ -997,6 +1069,52 @@ export class WebsocketClient extends EventEmitter {
     }
   }
 
+  private updatePendingTopicSubscriptionStatus(wsKey: string, msg: any) {
```

Comment:
Since exceptions here could be quite critical (they would prevent events from flowing to the consumer), I would suggest to use `unknown` as the type instead of `any`, forcing very strict (and somewhat annoying, I know, sorry) type checks in the assumptions you make about the type of `msg`. Same thing anywhere else the type is set to `any`. Happy to discuss if you're not sure about handling it!